### PR TITLE
feat: VC signing bddtest

### LINKED ISSUE #1342: VC signing bddtest
1- create did store it in sidetree-mock
2- sign vc using private key
3- resolve did and verify vc using public key

### PR DESCRIPTION
**Pull request by kdimak**
closes #1342

Signed-off-by: Dmitriy Kinoshenko <dkinoshenko@gmail.com>

VC signing and verification using JWS Linked Data proof.
JWS proof will be added as a follow-up PR (which should close the issue).

### CODE REVIEW

File: test/bdd/features/verifiable_credential.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@verifiable
+Feature: Issue Verifiable Credential
+  @issue_vc_ldp
+  Scenario: Issue University Degree Credential with JWS Linked Data proof
+    Given "Stanford University" issues credential at "2018-03-15" regarding "Bachelor Degree" to "Alice"
+    When "Stanford University" adds "JWS Linked data" proof and issues the credential
```
Comment (rolsonquadras) [inline test/bdd/features/verifiable_credential.feature]: To me, looks like these 2 steps can be combined into one (both steps Issues Credential) - `"Stanford University" issues "Bachelor Degree" credential at "2018-03-15" to "Alice" with "JWS Linked data" proof`.

Comment (sandrask) [reply test/bdd/features/verifiable_credential.feature]: this is true, you can add type at the end of first line e.g. with  "JWS Linked data" proof

Comment (kdimak) [reply test/bdd/features/verifiable_credential.feature]: Agree, I just wanted to split this into two lines for me readability.

Comment (sandrask) [reply test/bdd/features/verifiable_credential.feature]: It is more readable as Rolson suggested

Comment (kdimak) [reply test/bdd/features/verifiable_credential.feature]: Corrected. The code now looks more concise and readable, thanks @rolsonquadras !

**Pull request merged**